lib/shellwords.rb

```diff
@@ -68,6 +68,9 @@
 # 1: {IEEE Std 1003.1-2008, 2016 Edition, the Shell & Utilities volume}[http://pubs.opengroup.org/onlinepubs/9699919799/utilities/contents.html]
 
 module Shellwords
+
```

Comment:
Omit this blank line and start with a non-blank one.

Reply:
I missed this comment. I applied it at https://github.com/ruby/shellwords/commit/b45de514abcb293e9d8841c2a245a9178ee73280